src/services/sqs-service.ts

```diff
@@ -26,20 +28,34 @@ class SqsService {
     return SqsService._instance
   }
 
-  async sendMessage(sqsMessage: SqsMessageBody) {
+  async healthCheck() {
     try {
       const data = await this.client.send(
         new SendMessageCommand({
-          MessageBody: JSON.stringify(sqsMessage),
+          MessageBody: '{"eventName": "sample", "eventSignature": "sampleHandler", "msg":"hello sample body"}',
           QueueUrl: this.sqsQueueURL,
         }),
       )
       return data
     } catch (error: any) {
       console.log(error)
+      // eslint-disable-next-line no-process-exit, unicorn/no-process-exit
```

Comment:
Can we make a generic log message for this error. Maybe something like: "[Error] Indexer unable to reach queue system"? You can keep the log about the error.